src/utils/.eslintrc

```diff
@@ -0,0 +1,7 @@
+{
+  "rules": {
+    "import/no-extraneous-dependencies": [2, {
+      "devDependencies": true
```

Comment:
should these utils maybe live inside `scripts/`, since they're not for public consumption?

Reply:
well actually they might be? as part of the eventual initialize script.

Reply:
if that's the case then they definitely shouldn't be allowed to have dev deps :-)